submissions/alstep07/task-js-dom/app.js

```diff
@@ -0,0 +1,70 @@
+import { movies } from "./data.js";
+
+const main = document.querySelector('.main');
+const navBtn = document.querySelector('.navbar__list');
+const headerBtn = document.querySelector('.header__title');
+
+render(movies.all);
+
+headerBtn.addEventListener('click', function() {
+    render(movies.all);
+});
+
+navBtn.addEventListener('click', function(event){
+    render(movies[event.target.id]);
+});
+
+function render(genre){
+    
+    while (main.firstChild) {
+        main.removeChild(main.firstChild);
+    }
```

Comment:
[Not fixed](https://github.com/kottans/frontend-2021-homeworks/pull/245#discussion_r538598936)